### ISSUE TITLE
New: Rachmaninoff 10 Preludes Op23

### ISSUE THREAD

**Issue by dominicus** [MutopiaProject/MutopiaProject#354]
- [IMSLP Ref Page](http://imslp.org/wiki/10_Preludes,_Op.23_%28Rachmaninoff,_Sergei%29)
- [PDF](http://imslp.org/wiki/Special:ImagefromIndex/105592) - Gutheil, n.d.[1904].
- IMSLP Popularity (downloads/day): 80
- YearComposed/Style/Instrument: 1901  Romantic    Piano
- Comments: 48 pages.  Contact organizer if you wish to typeset only a subset of the preludes.
- [x]   1. Largo (F♯ minor)                    _Assigned to @abelcheung_
- [x]   2. Maestoso (B♭ major)              _Assigned to @abelcheung_
- [x]   3. Tempo di minuetto (D minor) _Assigned to @abelcheung_
- [x]   4. Andante cantabile (D major)  _Assigned to @abelcheung_
- [x]   5. Alla marcia (G minor)             _Assigned to @joram-berger_
- [x]   6. Andante (E♭ major)                _Assigned to @abelcheung_
- [ ]   7. Allegro (C minor)                    _Assigned to @joram-berger_
- [x]   8. Allegro vivace (A♭ major)        _Assigned to @glenl_
- [x]   9. Presto (E♭ minor)                   _Assigned to @joram-berger_
- [x] 10. Largo (G♭ major)                    Joshua Nichols josh.d.nichols@gmail.com

**Comment by joram-berger** [comment]
Is it ok to use includes from the openlilylib snippets repository or should the ly file(s) be self-contained and only require LilyPond? I ask because I like the edition-engraver there. In general, I see three options:
- put all the tweaks in the music
- add extra voices for tweaks
- use the edition engraver from oll
- copy the code from oll to the ly file
What do you prefer?

**Comment by dominicus** [comment]
@joram-berger I'm not familiar with the openlilylib edition-engraver.
For future maintainability, LilyPond-only code is best, as we rely on convert-ly to keep sources updated.
It's common for contributors to leverage LSR and openlilylib for snippets and scheme functions.
How you incorporate these is at your discretion.  Simple functions and overrides work fine in the score .ly source.  More complex functions like [\shapeII](https://github.com/openlilylib/openlilylib/blob/master/notation-snippets/shaping-bezier-curves/shapeII.ily) work better in a separate \include .ily for functions and definitions.
It's good practice to add a comment pointing to the source of the snippet.  This way, if we face issues updating the piece to a future LilyPond release using convert-ly, we have a way to check the source of the snippet for updates.

**Comment by joram-berger** [comment]
I could take Prelude 9 (Presto), if you guys like. I will start in case I get assigned to it.

**Comment by dominicus** [comment]
@joram-berger assigned.  Thank You.

**Comment by joram-berger** [comment]
No. 9 done in pull request [665](https://github.com/MutopiaProject/MutopiaProject/pull/665).

**Comment by glenl** [comment]
Please assign #8 Allegro Vivace to me if still unassigned.

**Comment by joram-berger** [comment]
I once contributed two of these preludes and just realized that the issue is still open.
@abelcheung will you still work on No. 7? If not I could engrave op. 23 No. 7. There might be some weekend for that...

**Comment by dominicus** [comment]
I followed up with @abelcheung on No7 a while back but didn't get a status.
No.7 now assigned to @joram-berger.
Thx for picking up the one outstanding from this collection.
